app/src/main/java/ca/chronofit/chrono/circuit/CircuitTimerActivity.kt

```diff
@@ -288,18 +285,31 @@ class CircuitTimerActivity : BaseActivity() {
                 bind.initButtonLayout.visibility = View.GONE
                 bind.runButtonLayout.visibility = View.VISIBLE
                 bind.pauseButtonLayout.visibility = View.GONE
+
+                bind.pauseButton.setTextColor(ContextCompat.getColor(this, R.color.lightText))
```

Comment:
Chave this is the XML, not here. The color is static so we don't need to keep calling it.

Reply:
English pls

Reply:
KK I fixed it